Makefile

```diff
@@ -119,6 +119,10 @@ install-envtest:
 integration-test: install-envtest
 	$(INTEGTESTENVVAR) hack/integration-test.sh
 
+.PHONY: integration-test-cache
+integration-test-cache: install-envtest
+	$(INTEGTESTENVVAR) hack/integration-test-cache.sh
```

Comment:
Hrm, why make it a separate test?

Reply:
it was the quickest (admittedly, did poor research before to take this route) to focus on the new tests and cut development time. Thanks for your suggestions below!